### PR TITLE
Bump to v1.0.0-beta.19

### PR DESCRIPTION
**Pull request by artonge**
Needed for a proper release.

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]
Ahahah

**Review by skjnldsv — CHANGES_REQUESTED** [review]
Wait, you can actually still do 19, npm have just refused it, no?
Just delete the tag, amend so it's beta 19 and push again?

**Comment by artonge** [comment]
Thought about it but was lazy.
Anyway, done :)

File: package.json [hunk 1]
```diff
@@ -94,4 +94,4 @@
   "peerDependencies": {
     "@nextcloud/vue": "^8.0.0-beta || ^8.0.0"
   }
-}
+}
```
Comment (skjnldsv) [inline package.json:97]:
```suggestion
}

```

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**